### ISSUE TITLE
The first edit box to change the nickname when open  preferences should be focused instead of a tab.

### ISSUE THREAD

**Issue by Ramon-Salazar** [BearWare/TeamTalk5#1788]
### Description

This is a weird bug since normally each dialog should show the first option when opening it.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

When opening the preferences it should show the first option in the general tab which is the edit box to add/change the nickname

### Actual behavior

It focuses on general tab, which isn't expected.

### Steps to reproduce problem

1: by pressing f4 or by right click on client/preferences. 2: the window will show up, focuses on the general tab instead of the first option

**Comment by CoBC** [comment]
No, this behavior was simply forced when I worked on focus placement in 
dialogs.

**Comment by Ramon-Salazar** [comment]
You're wrong, In classic client it focuses on the edit box to change the nickname, which is the first option in the general tab, while in QT it focuses on general tab.
